StreamChatClient_v3/APIClient/Endpoints/Payloads/ChannelCreatePayload.swift

```diff
@@ -0,0 +1,58 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+public struct ChannelCreatePayload<ExtraData: ExtraDataTypes>: Encodable {
```

Comment:
Why you decide to create a separate type instead of `ChannelDetailPayload`?

Reply:
I don't think it should be public.

Reply:
- Yes, this doesn't need to be public
- @buh , I think it's good to have a separate type for this. `ChannelDetailPayload` has fields, that can be modified only on the server (like `createdAt`, `lastMessageAt`, etc.). This type should represent only fields we're able to modify from the client. Btw - do you happen to know the complete list of them?
- The naming can be better, true. Maybe `ChannelEditPayload` since we will use it also for editing existing channels?

Reply:
Done